webview/src/experiments/components/table/styles.module.scss

```diff
@@ -166,12 +162,6 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
       border-bottom-color: $row-hover-background-color;
     }
 
-    .runningExperiment:not(.rowSelected) .experimentCell:hover & {
```

Comment:
High contrast light theme

<img width="138" alt="image" src="https://user-images.githubusercontent.com/37993418/185587129-7460305b-3fc2-4341-991f-ae7ef7164517.png">

Dive bar: 

<img width="197" alt="image" src="https://user-images.githubusercontent.com/37993418/185587275-1429bddc-118b-44eb-a05b-a68a48cde3fc.png">

Reply:
Figured out a way to fix the background, though the spinners border are still a slight different color. You can't really notice unless you squint, so should be good to merge :) 

<img width="263" alt="image" src="https://user-images.githubusercontent.com/43496356/185659461-d81edce8-f2f9-4d84-830e-a484c0c4e413.png">